From 0aaad55cf35d9e8af36b1644840725fcc07ca057 Mon Sep 17 00:00:00 2001
From: Eric Sandeen <sandeen@redhat.com>
Date: Mon, 4 Aug 2014 18:58:29 -0400
Subject: [fs] btrfs: remove unused variable from find_free_extent

Message-id: <1407179109-13714-23-git-send-email-sandeen@redhat.com>
Patchwork-id: 86970
O-Subject: [RHEL7.1 PATCH 022/422] btrfs: remove unused variable from find_free_extent
Bugzilla: 1071179
RH-Acked-by: Zach Brown <zab@redhat.com>

From: Valentina Giusti <valentina.giusti@microon.de>

commit 4b447bfac6c6b367a594a22cb220152f835fc0ed
Author: Valentina Giusti <valentina.giusti@microon.de>
Date:   Mon Nov 4 22:34:24 2013 +0100

    btrfs: remove unused variable from find_free_extent

    The variable found_uncached_bg in find_free_extent is not used since commit
    285ff5af6ce358e73f53b55c9efadd4335f4c2ff
    (Btrfs: remove the ideal caching code)

    Signed-off-by: Valentina Giusti <valentina.giusti@microon.de>
    Signed-off-by: Josef Bacik <jbacik@fusionio.com>
    Signed-off-by: Chris Mason <clm@fb.com>
Signed-off-by: Eric Sandeen <sandeen@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/fs/btrfs/extent-tree.c b/fs/btrfs/extent-tree.c
index e094d02..fe651f4 100644
--- a/fs/btrfs/extent-tree.c
+++ b/fs/btrfs/extent-tree.c
@@ -6239,7 +6239,6 @@ static noinline int find_free_extent(struct btrfs_root *orig_root,
  int index = __get_raid_index(flags);
  int alloc_type = (flags & BTRFS_BLOCK_GROUP_DATA) ?
   RESERVE_ALLOC_NO_ACCOUNT : RESERVE_ALLOC;
- bool found_uncached_bg = false;
  bool failed_cluster_refill = false;
  bool failed_alloc = false;
  bool use_cluster = true;
@@ -6357,7 +6356,6 @@ search:
 have_block_group:
   cached = block_group_cache_done(block_group);
   if (unlikely(!cached)) {
-   found_uncached_bg = true;
    ret = cache_block_group(block_group, 0);
    BUG_ON(ret < 0);
    ret = 0;
-- 
1.7.1